drivers/CoreLED/tests/RGB_test.cpp

```diff
@@ -0,0 +1,23 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "RGB.h"
+
+#include "gtest/gtest.h"
+
+using namespace leka;
+
+TEST(RGBTest, equalOperator)
+{
+	RGB black = RGB::black;
+	RGB gold  = RGB {255, 215, 00};
+
+	bool isEqual1 = (black == RGB::black);
+	bool isEqual2 = (black == RGB {0x00, 0x00, 0x00});
+	bool isEqual3 = (gold == RGB {255, 215, 0});
+
+	ASSERT_EQ(isEqual1, true);
+	ASSERT_EQ(isEqual2, true);
+	ASSERT_EQ(isEqual3, true);
```

Comment:
Tu peux utiliser directement `ASSERT_TRUE` (et/ou `ASSERT_FALSE`) dans le cas d'un bool :)

```suggestion
	bool isEqual1 = (black == RGB::black);
	bool isEqual2 = (black == RGB {0x00, 0x00, 0x00});
	bool isEqual3 = (gold == RGB {255, 215, 0});

	ASSERT_TRUE(isEqual1);
	ASSERT_TRUE(isEqual2);
	ASSERT_TRUE(isEqual3);

	bool isEqual4 = (gold == RGB {0x00, 0x00, 0x00});

	ASSERT_FALSE(isEqual4);
```